bridges/modules/xcm-bridge-router/src/lib.rs

```diff
@@ -133,11 +129,6 @@ pub mod pallet {
 
 	#[pallet::config(with_default)]
 	pub trait Config<I: 'static = ()>: frame_system::Config {
-		/// The overarching event type.
-		#[allow(deprecated)]
-		#[pallet::no_default_bounds]
-		type RuntimeEvent: From<Event<Self, I>>
```

Comment:
@rosarp yes, you're right and I was wrong before (more context: https://github.com/paritytech/polkadot-sdk/pull/7229), let's remove :)